src/csharp/experimental/build_native_ext_for_android.sh

```diff
@@ -15,6 +15,8 @@
 
 # Helper script to crosscompile grpc_csharp_ext native extension for Android.
 
+set -ex
+
```

Comment:
nit that extends beyond this PR: speed up make -j4, or 8?

Reply:
done